src/components/Accounts/ViewFeesFines/ViewFeesFines.js

```diff
@@ -190,6 +190,12 @@ class ViewFeesFines extends React.Component {
     return res;
   }
 
+  formatTitle(f) {
```

Comment:
Could you please clarify what does it mean 'f' in this case? 

This approach is used at this file, but it seems we can try to add more clear names in scope of new changes. @ViktorSoroka What do you think about it?